libs/clients/ums-cost-of-living-calculator/src/lib/fetch/apis/CostOfLivingCalculatorApi.ts

```diff
@@ -0,0 +1,47 @@
+import { Inject, Injectable } from '@nestjs/common'
+import { UmbodsmadurSkuldaraClientConfig } from '../../umsCostOfLivingCalculator.config'
+import { ConfigType } from '@nestjs/config'
+export interface CostOfLivingCalculatorRequest {
+  fjMerking: string
+  fot: number
+  laek: number
+  matur: number
+  onnurthon: number
+  samgongur: number
+  samskipti: number
+  samtals: number
+  text: string
+  tom: number
+}
+@Injectable()
+export class CostOfLivingCalculatorApi {
+  constructor(
+    @Inject(UmbodsmadurSkuldaraClientConfig.KEY)
+    private clientConfig: ConfigType<typeof UmbodsmadurSkuldaraClientConfig>,
+  ) {}
+  async getCalculatorData(): Promise<Array<CostOfLivingCalculatorRequest>> {
+    const response = await Promise.all([
+      fetch(this.clientConfig.url, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json; charset=utf-8',
+        },
+        body: JSON.stringify({ isEinstaklingur: 'false' }),
+      }),
+      fetch(this.clientConfig.url, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json; charset=utf-8',
+        },
+        body: JSON.stringify({ isEinstaklingur: 'true' }),
+      }),
+    ])
+
+    const data1 = await response[0].json()
```

Comment:
You could parhaps do

const [data1, data2] = await Promise.all(response.map(response => response.json()))

Or

add a .then behind each fetch that'll convert it to .json()